api/v1beta1/bgppeer_types.go

```diff
@@ -64,11 +62,11 @@ type BGPPeerSpec struct {
 
 	// Requested BGP hold time, per RFC4271.
 	// +optional
-	HoldTime time.Duration `json:"holdTime,omitempty" yaml:"hold-time,omitempty"`
+	HoldTime string `json:"holdTime,omitempty" yaml:"hold-time,omitempty"`
```

Comment:
I did a bit of research, and I think the right type to use here is metav1.Duration.
It will render as a string but it will be subject to the time validation. See https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration